Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -736,6 +736,14 @@ bool BBQJIT::canTierUpToOMG() const
     return true;
 }
 
+void BBQJIT::emitIncrementCallSlotCount(unsigned callSlotIndex)
+{
+    // FIXME: We should extend BBQ to have callee save registers and put this CallSlots vector to one of that register.
+    // This can reduce code size.
+    auto& slot = m_profiledCallee.callSlots()[callSlotIndex];
+    m_jit.add32(TrustedImm32(1), CCallHelpers::AbsoluteAddress(slot.addressOfCount()));
```

Comment:
We should probably note that this is knowingly racy but that's ok because it's just for profiling information, etc.